common/metrics/metric_defs.go

```diff
@@ -911,6 +911,7 @@ var (
 	ShardLingerSuccess                             = NewTimerDef("shard_linger_success")
 	ShardLingerTimeouts                            = NewCounterDef("shard_linger_timeouts")
 	DynamicRateLimiterMultiplier                   = NewGaugeDef("dynamic_rate_limit_multiplier")
+	DLQWrites                                      = NewCounterDef("dlq_writes")
```

Comment:
Could you add a description to this with the `WithDescription` option?

Reply:
Done